.github/workflows/make_bundle_conda.yml

```diff
@@ -0,0 +1,359 @@
+on:
+  push:
+    # Sequence of patterns matched against refs/tags
+    tags:
+      - "v*" # Push events to matching v*, i.e. v1.0, v20.15.10
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+  schedule:
+    - cron: "0 0 * * *"
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+name: Conda
+
+jobs:
+  conda-build:
+    name: Create packages
+    runs-on: ubuntu-latest
+    if: github.repository == 'napari/napari'
+    env:
+      conda_bld_suffix: /napari-feedstock/build_artifacts/
+    outputs:
+      artifact-id: ${{ steps.artifact-id.outputs.name }}
+
+    steps:
+      - name: Checkout napari source
+        uses: actions/checkout@v2
+        with:
+          path: napari-source
+          fetch-depth: 0
+
+      - name: Checkout conda-forge feedstock
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+          # TODO: Match upstream, not fork!
+          repository: jaimergp/napari-feedstock
```

Comment:
We need to [merge this PR](https://github.com/conda-forge/napari-feedstock/pull/32) to revert this. We can keep it open for a bit and I will sync it manually but ideally at some point this will be upstream.

Reply:
So, is that ready to be merged? 😊

Reply:
We can't merge it as is because that fork points to this branch. But the proposed changes should be added in the next release.